etc/ProdOfflineConfiguration.py

```diff
@@ -312,6 +312,31 @@
                  diskNode="T2_CH_CERN",
                  data_tiers=["FEVTHLTALL"],
                  write_dqm=True,
+                 alca_producers=["TkAlHLTTracks", "TkAlHLTTracksZMuMu", "PromptCalibProdSiPixelAliHLTHGC"],
+                 dqm_sequences=["@HLTMon"],
+                 reco_version=defaultCMSSWVersion,
+                 multicore=numberOfCores,
+                 global_tag_connect=globalTagConnect,
+                 global_tag=expressGlobalTag,
+                 proc_ver=expressProcVersion,
+                 maxInputRate=23 * 1000,
+                 maxInputEvents=400,
+                 maxInputSize=2 * 1024 * 1024 * 1024,
+                 maxInputFiles=15,
+                 maxLatency=15 * 23,
+                 periodicHarvestInterval=20 * 60,
+                 blockCloseDelay=1200,
+                 timePerEvent=4, #I have to get some stats to set this properly
+                 sizePerEvent=1700, #I have to get some stats to set this properly
+                 maxMemoryperCore=2000,
+                 dataset_lifetime=3*30*24*3600,#lifetime for container rules. Default 3 months
+                 versionOverride=expressVersionOverride)
+
+addExpressConfig(tier0Config, "CosmicsHLTMonitor",
```

Comment:
@LinaresToine I just realized that there is yet another mistake here.
The name of the PD is `CosmicHLTMonitor`  and not `CosmicsHLTMonitor` (without an "s"), see the description from @pietroGru at in the [Tier0 ops cmsTalk](https://cms-talk.web.cern.ch/t/new-primary-dataset-cosmichltmonitor-for-cosmics-hlt-menus/118912/1)